src/util/checkIfPathShareSubcontext.js

```diff
@@ -0,0 +1,20 @@
+import { equalThoughtRanked } from '../util'
+
+export const checkIfPathShareSubcontext = (rankedThoughtsA, rankedThoughtsB) => {
+  /* this method returns -1 if there is no common majority subcontext else returns the index up to which two rankedThought share common majority subcontext */
```

Comment:
Use `//` single-line comments for regular comments, `/*` block comments for JSDOC and comments longer than 2 lines.

Reply:
Sure